app/Components/AddForm.tsx

```diff
@@ -0,0 +1,317 @@
+"use client";
+import React, { ChangeEvent, FormEvent } from "react";
+import { useState } from "react";
+import { createEmployee } from "@/lib/createEmployee";
+
+const Addform = () => {
+  const [newEmployee, setNewEmployee] = useState<Employee>({
+    accountName: "",
+    _id: "",
```

Comment:
This field is not needed while adding an employee

Reply:
The id is needed for the sub route since I'm getting an employee by his id.

Reply:
Submitting the form without account name returns 400 status

Reply:
Effected..